test/server/cards/03_TWI/events/NowThereAreTwoOfThem.spec.ts

```diff
@@ -0,0 +1,69 @@
+describe('Now There are Two of Them', function() {
+    integration(function(contextRef) {
+        describe('Now There are Two of Them\'s ability -', function() {
+            it('should allow you to play a non-Vehicle unit from your hand that shares a Trait with the unit you control for 5 less, if you have only one unit in play', function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['now-there-are-two-of-them', 'zuckuss#bounty-hunter-for-hire', 'cad-bane#hostage-taker', 'plo-koon#kohtoyah', 'fetts-firespray#pursuing-the-bounty'],
+                        groundArena: ['greedo#slow-on-the-draw'],
+                        base: { card: 'administrators-tower', damage: 0 },
+                        leader: 'qira#i-alone-survived',
+                        resources: 5
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // You can select Zuckuss and Cad Bane. Zuckuss is free to play.
```

Comment:
Could we play something that costs at least 6 so we can confirm that the discount amount is right